projects/train/configs/bns.yaml

```diff
@@ -0,0 +1,102 @@
+# commented args represent values filled out
+# by train task at run time. To build a functional
+# standalone config, add these in.
+ 
+# To start training from a checkpoint, uncomment the below argument
+# and specify a path to the desired checkpoint
+# ckpt_path: ""
+matmul_precision: "highest"
+model:
+  class_path: train.model.SupervisedAframe
+  init_args:
+    # architecture
+    arch:
+      class_path: train.architectures.supervised.SupervisedFrequencyDomainResNet
+      init_args:
+        layers: [3, 4, 6, 3]
+        norm_layer:
+          class_path: ml4gw.nn.norm.GroupNorm2DGetter
+          init_args:
+            groups: 16
+    metric:
+      class_path: train.metrics.TimeSlideAUROC
+      init_args:
+        max_fpr: 1e-3
+        pool_length: 8
+
+    # optimization params
+    weight_decay: 0.0
+    learning_rate: 0.000585
+    pct_lr_ramp: 0.115
+    # early stop
+    patience: 50
+data:
+  class_path: train.data.supervised.FrequencyDomainSupervisedAframeDataset
+  init_args:
+    # loading args
+    # data_dir:
+    # ifos:
+
+    # preprocessing args
+    batch_size: 576
+    # kernel_length:
+    psd_length: 8
+    # fduration:
+    # highpass: 
+    fftlength: null
+
+    # augmentation args
+    waveform_prob: 0.277
+    swap_prob: 0.014
+    mute_prob: 0.055
+    trigger_pad: -0.75
+    snr_sampler:
+      class_path: ml4gw.distributions.PowerLaw
+      init_args: 
+        minimum: 10
+        maximum: 100
+        index: -3
+    qtransform:
+      class_path: ml4gw.transforms.SingleQTransform
+      init_args:
+        duration: 8
```

Comment:
@EthanMarx Is there a way to have this value use the value for `kernel_length` given in the sandbox config? I.e., can multiple arguments in this config be taken from a single argument in the sandbox config?

Reply:
The way to do this would be via [argument linking](https://lightning.ai/docs/pytorch/stable/cli/lightning_cli_expert.html#argument-linking) in the LightningCLI parser. 

Since this won't be needed in the BBH case, might make sense to subclass the CLI for this.